packages/yoga/src/AutoComplete/web/AutoComplete.jsx

```diff
@@ -1,9 +1,10 @@
 /* eslint react/no-array-index-key: 0 */
-import React, { useState } from 'react';
+import React, { useState, useCallback, useRef, useEffect } from 'react';
 import Downshift from 'downshift';
 import { arrayOf, string, func, bool, shape } from 'prop-types';
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 
+import { ChevronDown, ChevronUp } from '@gympass/yoga-icons/src';
```

Comment:
Why do we need to get it from the /src?
```suggestion
import { ChevronDown, ChevronUp } from '@gympass/yoga-icons';
```

Reply:
It's an error from the VSCode auto import 😕 fixed!